### DISCUSSION TITLE
Sir, may I join the team ?

### DISCUSSION THREAD

**Discussion by mitmac29** [ROM-builders/temporary#19514] [General]
GitHub : mitmac29
Telegram : @mitmac29

Device tree : https://github.com/mitmac29/android_device_xiaomi_lavender/tree/pixel13

Successful build : https://cirrus-ci.com/build/5785199800221696

**Comment by Apon77** [comment]
telegram and GitHub username mismatches

**Reply by mitmac29** [reply]
> telegram and GitHub username mismatches
@Apon77 
Sir, I've changed it as it should be ... please have a look again ...
